modules/create_timepoint/php/create_timepoint.class.inc

```diff
@@ -219,5 +228,31 @@ class Create_Timepoint extends \NDB_Form
 
         return $errors;
     }
+
+    /**
+     * Generate a breadcrumb trail for that page.
+     *
+     * @return \LORIS\BreadcrumbTrail The trail.
```

Comment:
I don't know if the description adds much value here, you can probably just define the return type.

Reply:
phpcs only requires the return type declaration, not the description after the type.